modules/tinymce/src/core/test/ts/browser/EditorManagerTest.ts

```diff
@@ -238,4 +241,48 @@ describe('browser.tinymce.core.EditorManagerTest', () => {
 
     assert.notEqual(editor1.editorUid, editor2.editorUid);
   });
+
+  context('locked properties', () => {
```

Comment:
Could we split this up into two contexts. One for the editor and one for the editor manager or even move the second one into some Editor centric test.

Reply:
I have refactored the tests 👍